src/cfnlint/__init__.py

```diff
@@ -38,8 +38,10 @@ class CloudFormationLintRule(object):
     tags = []
```

Comment:
this might have the same problem?

Reply:
It could.  We have been specifying the tags in the attributes of the sub classes.  If anyone was doing the population using the __init__ this would probably pop up.  I suppose the prudent thing would be to fix it before that happens.